connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -198,6 +199,8 @@ private Long ensureLeaderConfig(long maxOffset) {
             }
         }
 
+        this.leaderState = new LeaderState(allConfigs, connectorAssignments, taskAssignments);
```

Comment:
Good point. It does get exposed, if indirectly, through the status endpoint, but it might make sense to have a separate endpoint.